src/entities/address-entity.ts

```diff
@@ -0,0 +1,32 @@
+import { Entity, Column, PrimaryColumn, ManyToOne } from "typeorm";
+import { User } from "./user.entity";
+
+@Entity({ name: "address" })
+export class Address {
+  @PrimaryColumn()
+  CEP: string;
+
+  @Column()
+  street: string;
+
+  @Column()
+  streetNumber: number;
+
+  @Column({ nullable: true })
+  complement: string;
```

Comment:
sem o `?` na classe ele fica opcional no banco mas obrigatório no TS
```suggestion
  @Column({ nullable: true })
  complement?: string;
```